.github/workflows/static_build.yml

```diff
@@ -13,58 +13,64 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  micromamba_full_static:
-    runs-on: ${{ matrix.os }}
+  conda-build:
+    name: "micromamba - ${{ matrix.TARGET }}"
+    runs-on: ${{ matrix.OS }}
+    env:
+      CI: True
+      CONDA_BUILD_YML: ${{ matrix.CONDA_BUILD_YML }}
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        include:
+          - { OS: ubuntu-latest, CONDA_BUILD_YML: linux_64_, TARGET: linux-64, BUILD: linux-64 }
```

Comment:
In GHA it's convention to use `kebab-case`.

In conda-forge terms, `BUILD` = `host`.

Reply:
Would either include the full file name in `CONDA_BUILD_YML` so it's clear a it's a file name: `conda-build-yml-file: linux_64_.yaml`. Or autogenerate from the `target`: `echo ${{ matrix.target }}_ | tr _ -`.